src/main/java/org/cryptomator/ui/error/ErrorController.java

```diff
@@ -65,11 +66,13 @@ public class ErrorController implements FxController {
 	private final Scene previousScene;
 	private final Stage window;
 	private final Environment environment;
+	private final ExecutorService executorService;
 
 	private final BooleanProperty copiedDetails = new SimpleBooleanProperty();
 	private final ObjectProperty<ErrorDiscussion> matchingErrorDiscussion = new SimpleObjectProperty<>();
 	private final BooleanExpression errorSolutionFound = matchingErrorDiscussion.isNotNull();
 	private final BooleanProperty isLoadingHttpResponse = new SimpleBooleanProperty();
+	private final BooleanProperty lookupDatabaseUserPermission = new SimpleBooleanProperty();
```

Comment:
Variable name is confusing. It does not represent the decision the user made, as "permission" in combination with boolean suggests, but only, if the user clicked one of the buttons.

How about `ShowingDatabaseLookupPermission" ?

Reply:
renamed to askedForLookupDatabasePermission